modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/cluster/services/Cluster.scala

```diff
@@ -44,15 +45,19 @@ object Cluster {
         for {
           session <- sessionStorage.getToken.flatMap {
             case Some(s) => Applicative[F].pure(s)
-            case None    => MonadThrow[F].raiseError[SessionToken](SessionDoesNotExist)
+            case None =>
```

Comment:
Looks like you changed the format unexpectedly

Reply:
Reverted this change.